cms/lib/ai_aside_summary_config.py

```diff
@@ -0,0 +1,54 @@
+"""
+This file contains AiAsideSummaryConfig class that take a `course_key` and return if:
+    * the waffle flag is enabled in ai_aside
+    * is the summary is enabled for a given unit_key
+    * change the settings for a given unit_key
+"""
+
+
+class AiAsideSummaryConfig:
+    """
+    Configuration for the AI Aside summary configuration.
+    """
+
+    def __init__(self, course_key):
+        self.course_key = course_key
+
+    def __str__(self):
+        """
+        Return user-friendly string.
+        """
+        return f"AIAside summary configuration for {self.course_key} course"
+
+    def is_enabled(self):
+        """
+        Define if the waffle flag is enabled for the current course_key
```

Comment:
this doc should mention that if ai_aside is not installed it does not error but instead returns false, that's really critical behavior for this object

and the same thing below

Reply:
Adding this, good catch 👍🏽